test/infrastructure/docker/internal/controllers/dockercluster_controller.go

```diff
@@ -188,18 +186,18 @@ func (r *DockerClusterReconciler) reconcileDelete(ctx context.Context, dockerClu
 }
 
 // SetupWithManager will add watches for this controller.
-func (r *DockerClusterReconciler) SetupWithManager(mgr ctrl.Manager, options controller.Options) error {
+func (r *DockerClusterReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, options controller.Options) error {
```

Comment:
If we are changing this method signature, I'm expecting the call to this method to change as well..